rubocop-airbnb/lib/rubocop-airbnb.rb

```diff
@@ -4,6 +4,9 @@
 # Load original rubocop gem
 require 'rubocop'
 
+require 'rubocop-performance'
+require 'rubocop-rails'
```

Comment:
With this addition, shouldn't it mean that this commit is no longer necessary? 72066c174f4f5c3750c51bebf711ca831df5b83a